media-libs/libopenshot/libopenshot-0.1.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Video editing library used by OpenShot"
+HOMEPAGE="http://www.openshotvideo.com/"
+SRC_URI="https://launchpad.net/${PN}/$(get_version_component_range 1-2)/${PV}/+download/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="libav"
+
+RDEPEND="
+	media-libs/libopenshot-audio
+	dev-qt/qtmultimedia:5[widgets]
+	media-gfx/imagemagick
+	libav? ( media-video/libav:=[encode,x264,xvid,vpx,mp3,theora] )
+	!libav? ( media-video/ffmpeg:0=[encode,x264,xvid,vpx,mp3,theora] )
+"
+DEPEND="
+	dev-lang/swig
```

Comment:
It looks like swig is only required when building the python and/or ruby bindings. These bindings are currently built automagically, so they would probably need some eclass support and USE flags.

Alternatively, if the bindings aren't required right now they could be disabled entirely.